### ISSUE TITLE
Dev and test set overlap in >10k examples?

### ISSUE THREAD

**Issue by gregor-ge** [virginie-do/e-SNLI-VE#4]
Hello,

I noticed by accident that identical images appear in both the dev and the test set.
I then discovered that the two sets overlap in 11896 examples, that is ~2/3 of all examples! This seems not correct to me.
Did you maybe make a mistake in the uploaded files?

A few examples that appear in both sets: '3575971480.jpg#1r2e', '4650367147.jpg#0r1e', '250346988.jpg#0r1e', '1235681222.jpg#3r5c', '4811454472.jpg#4r1e', '4869914617.jpg#4r1c', '2984704498.jpg#2r1c', '2678193588.jpg#0r4c', '86112925.jpg#4r1c', '2988244398.jpg#3r1e

The training set does not overlap with the other sets.
